source/shared/cpp/ObjectModel/Column.cpp

```diff
@@ -95,7 +95,8 @@ void Column::GetResourceInformation(std::vector<RemoteResourceInformation>& reso
 void Column::DeserializeChildren(ParseContext& context, const Json::Value& value)
 {
     // Parse Items
-    auto cardElements = ParseUtil::GetElementCollection(context, value, AdaptiveCardSchemaKey::Items, false);
+    auto cardElements =
+        ParseUtil::GetElementCollection<BaseCardElement>(true, context, value, AdaptiveCardSchemaKey::Items, false);
```

Comment:
NIt: Nice to include the argument names in comments /**/ style for readability

Reply:
Personally, I tend to find it harder to read to have the comments mixed in inline like that. That said, i see that those bools are non-obvious if you're in code flow and not an IDE. Took a look at the coding guidelines, which prefer // comments over /**/ comments, and went with separating out the parameters and adding end of line comments to the bools.

---
In reply to: [279800461](https://github.com/Microsoft/AdaptiveCards/pull/2802#discussion_r279800461) [](ancestors = 279800461)